README.md

```diff
@@ -33,94 +33,94 @@ I've written an introductory post, explaining [what mqttwarn can be used for](ht
     + [Producing JSON](#producing-json)
   * [Notes](#notes)
   * [Press](#press)
-  
-      
+
```

Comment:
Did you apply some sort of auto-formatting to this file?  There are a LOT of extraneous changes, making it very difficult to find any actual changes.  Please undo any unnecessary change.